content/tutorials/r_gbif_name_matching/index.Rmd

```diff
@@ -12,70 +12,92 @@ output:
 
 ## Introduction
 
-This tutorial will explain how you can match a list of 'scientific names' to the GBIF taxonomic backbone
+This tutorial will explain how you can match a list of _scientific names_ against the [GBIF backbone taxonomy](https://www.gbif.org/dataset/d7dddbf4-2cf0-4f39-9b2a-bb099caae36c).
 
-Important is that you have [rgbif](https://github.com/ropensci/rgbif) and [inborutils](https://inbo.github.io/inborutils/) installed and available:
+Important is that you have the most recent version of [inborutils](https://inbo.github.io/inborutils/) installed and available:
```

Comment:
Change to: It is important that ...

Reply:
See https://github.com/inbo/tutorials/pull/146#discussion_r316063029